python/lsst/verify/measurement.py

```diff
@@ -249,7 +249,11 @@ def _repr_latex_(self):
         rep : `str`
             String representation.
         """
-        return '{0.value:0.1f} {0.unit:latex_inline}'.format(self.quantity)
+        # If the value is less than 0.01, represent in scientific notation
+        if (self.quantity.value < 1e-2):
+            return '{0.value:0.2e} {0.unit:latex_inline}'.format(self.quantity)
```

Comment:
If this is supposed to be LaTeX-formatted, do you need to explicitly handle the `A×10^{B}`? I haven't worked with LaTeX in a few years, but I think I remember it being really unfriendly to `AeB` notation.

Reply:
In my tests, this seems to work fine.